fhir-path/src/main/java/com/ibm/fhir/path/FHIRPathParser.java

```diff
@@ -1,3 +1,9 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 // Generated from FHIRPath.g4 by ANTLR 4.9.1
```

Comment:
this is a generated file, so i'm not sure if the header is needed or not.  i wonder if there's a way to make antlr generate it...